moped-database/metadata/databases/default/tables/public_moped_note_types.yaml

```diff
@@ -8,6 +8,7 @@ select_permissions:
         - id
         - name
         - slug
+        - source
```

Comment:
this helps prevent rendering UI elements that are rendered from the rows in `moped_note_types` like the radio buttons in the rich text editor (`source = ecapris`).